src/Hl7.Fhir.Support.Poco/Introspection/ModelInspector.cs

```diff
@@ -42,7 +41,7 @@ public class ModelInspector : IStructureDefinitionSummaryProvider
         /// metadata for the most recent release for those common classes.</remarks>
         public static ModelInspector ForAssembly(Assembly a)
         {
-            return _inspectedAssemblies.GetOrAdd(a.FullName, _ => configureInspector(a));
+            return _inspectedAssemblies.GetOrAdd(a.GetName().FullName, _ => configureInspector(a));
```

Comment:
Sure? Is that the same?

Reply:
I reversed the the change, and throw an `ArgumentNullException` when `a.FullName` is null.